with-baseweb/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+
+/.cache
+/.idea
```

Comment:
This should go in your own global `.gitignore`
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

```suggestion
```